Plist2swift/Plist2swift.swift

```diff
@@ -83,6 +83,69 @@ private func generateHeader() {
 		""")
 }
 
+/**
+Checking the key is present in the Optional Dictionary
+*/
+private func isKeyPresentInOptionalDictionary(keyToSearch: String, tupleKey: String, optionalDictionary: [String: [String: String]]) -> Bool {
+	guard let optionalKeysAndTypes = optionalDictionary[keyToSearch] else {
+		return false
+	}
+
+	let optionalArray = optionalKeysAndTypes.keys
+	return optionalArray.contains(tupleKey)
+}
+
+/**
+Checking the key is present in  all the plists
+*/
+private func isKeyAvailableInAllPlists(keyToSearch: String, tupleKey: String, tuplesForPlists: [String: KeyValueTuples]) -> Bool {
+	for plistPath in tuplesForPlists.keys {
```

Comment:
Please remove or extend the comment as it currently doesn’t add more information than the function signature.

Reply:
Removed